terraso_backend/apps/story_map/forms.py

```diff
@@ -0,0 +1,39 @@
+# Copyright © 2021-2023 Technology Matters
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see https://www.gnu.org/licenses/.
+
+import mimetypes
+
+import structlog
+from django import forms
+
+from .models import StoryMap
+
+mimetypes.init()
+logger = structlog.get_logger(__name__)
+
+
+class StoryMapForm(forms.ModelForm):
+    title = forms.CharField(max_length=128, required=True)
```

Comment:
question: should we allow longer titles?

Reply:
@paulschreiber Title is used to create the slug, so it seems reasonable that it has a limit because it will be used as an identifier but we can increase to what value do you think we should increase it?

Reply:
Ask @DerekCaelin for some sample storymap titles so we can see what is commonly used.

Reply:
Some examples:

**Title:** The Mud That Makes Us
**Subtitle:** An Extraordinary Earth Engine Shaping and Reshaping Guyana’s Coast

**Title:** Protecting Arid Ecosystems in Populated Areas
Link: https://unep-wcmc.maps.arcgis.com/apps/Cascade/index.html?appid=87e9f69d64e349728041200929281aaa

**Title:** Paisajes de "otro mundo" en España
**Subtitle:** De los farallones de Las Médulas, en León, al desierto aragonés de Los Monegros, escenarios alucinantes que se pueden visitar 
https://marketing.maps.arcgis.com/apps/MapTour/index.html?appid=44db58a1c0704e2e9b8e383a0a16b410

Reply:
@paulschreiber Those titles seem small enough, what do you think Paul?